src/components/routes.js

```diff
@@ -8,6 +8,6 @@ export default [
   { path: '/', component: HomePage, exact: true },
   { path: '/about', component: AboutPage },
```

Comment:
What's about to use the routes you have defined as a constants in src/constants/routes.js ?